static/app/components/updatedEmptyState.tsx

```diff
@@ -180,6 +188,15 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
                               )
                             ) : null}
                           </CodeSnippetWrapper>
+                          <CodeSnippetWrapper>
+                            {configuration.configurations ? (
+                              Array.isArray(configuration.configurations[0].code) ? (
```

Comment:
Can `configurations` be an array with length 0? If so this would error out

Reply:
shouldn't be possible - but ill add an extra check to be safe